modules/admin_manual/pages/troubleshooting/providing_logs_and_config_files.adoc

```diff
@@ -2,32 +2,28 @@
 :toc: right
 :page-aliases: configuration/server/logging/providing_logs_and_config_files.adoc
 
+:description: Generating configreports and prepare log files are essential to support identifying the cause of a problem. Here are instructions for how to collect them.
+
 == Introduction
 
-When you report a problem to {oc-support-url}[ownCloud Support] or our {oc-central-url}/latest[Forum (ownCloud Central)] you will be asked to provide certain log files or configurations for our engineers (or other users). 
-These are essential in better understanding your issue, your specific configuration, and the cause of the problem.
+{description}
 
-Here are instructions for how to collect them.
+When you report a problem to {oc-support-url}[ownCloud Support] or our {oc-central-url}/latest[Forum (ownCloud Central)] you will be asked to provide certain log files or configurations for our engineers or other supporting staff. This information is necessary in better understanding your issue, your specific configuration, and for further support.
 
 == Generate a Config Report
 
-You can use the webUI or the command line to generate a config report. 
-The webUI includes the web server environment, while the command line generated one doesn't as it can't access it.
-Therefore, if possible, always generate it through the webUI.
-Please note that you have to have the configreport app enabled. 
-Check if it's already enabled by going to the apps section of the admin settings.
-You can enable this app using the following commands:
+You can use the webUI or the command line to generate a config report. The webUI includes the web server environment, while the command line generated one doesn't as it can't access it. Therefore, if possible, always generate it through the webUI. The configreport app is always part of bundled apps, but it may needs to be enabled before it can be used.
```

Comment:
```suggestion
You can use the webUI or the command line to generate a config report. The webUI includes the web server environment, while the command line generated one doesn't as it can't access it. Therefore, if possible, always generate it through the webUI. The configreport app is always part of bundled apps, but it may need to be enabled before it can be used.
```